src/core/EffectChain.cpp

```diff
@@ -103,7 +103,8 @@ void EffectChain::loadSettings( const QDomElement & _this )
 			else
 			{
 				delete e;
-				e = new DummyEffect( parentModel(), effectData );
+				Engine * engine = Engine::inst();
```

Comment:
Was this part missed in the transition (it uses `Engine::<static methods>` instead of inheriting EngineClient)? Or is this intentional?